src/ztc/datamodel/admin/forms.py

```diff
@@ -2,11 +2,13 @@
 from django.utils.translation import ugettext_lazy as _
 
 import requests
+from vng_api_common.validators import ResourceValidator
```

Comment:
Please us [isort](https://pypi.org/project/isort/) to ensure imports are correctly sorted.

My shell aliases:

```bash
alias runisort='isort --recursive --check-only --diff .'
alias applyisort='isort --recursive .'
```